app/controllers/non_hmis_clients_controller.rb

```diff
@@ -94,9 +100,14 @@ def sorter
     return sort_string
   end
 
+  def build_assessment
+    assessment_type.constantize.new
```

Comment:
where does `assessment_type` come from?  It will need to be from a whitelist if we're going to call `constantize`.

Reply:
Found it.